project3/index.html

```diff
@@ -0,0 +1,251 @@
+                            fill="#fff" />
+                        <g stroke-width=".565" fill="#fff" stroke="#fff">
+                            <path
+                                d="M61.767 27.286c.1.1.3.2.5.2s.4-.1.5-.2c.3-.3.3-.8 0-1l-1.8-1.8c-.3-.3-.8-.3-1 0-.3.3-.3.8 0 1l1.8 1.8zM67.467 33.086c.1.1.3.2.5.2s.4-.1.5-.2c.3-.3.3-.8 0-1l-1.8-1.8c-.3-.3-.8-.3-1 0-.2.3-.3.8 0 1l1.8 1.8zM60.467 33.286c.2 0 .4-.1.5-.2l1.8-1.8c.3-.3.3-.8 0-1-.3-.3-.8-.3-1 0l-1.8 1.7c-.3.3-.3.8 0 1 .1.2.3.3.5.3zM66.267 27.486c.2 0 .4-.1.5-.2l1.8-1.8c.3-.3.3-.8 0-1-.3-.2-.8-.3-1 0l-1.8 1.8c-.3.3-.3.8 0 1 .1.1.3.2.5.2z"
+                                transform="matrix(.88398 0 0 .88643 -4.81 -5.506)" />
+                        </g>
+                        <g stroke-width=".565" fill="#fff" stroke="#fff">
+                            <path
+                                d="M61.767 27.286c.1.1.3.2.5.2s.4-.1.5-.2c.3-.3.3-.8 0-1l-1.8-1.8c-.3-.3-.8-.3-1 0-.3.3-.3.8 0 1l1.8 1.8zM67.467 33.086c.1.1.3.2.5.2s.4-.1.5-.2c.3-.3.3-.8 0-1l-1.8-1.8c-.3-.3-.8-.3-1 0-.2.3-.3.8 0 1l1.8 1.8zM60.467 33.286c.2 0 .4-.1.5-.2l1.8-1.8c.3-.3.3-.8 0-1-.3-.3-.8-.3-1 0l-1.8 1.7c-.3.3-.3.8 0 1 .1.2.3.3.5.3zM66.267 27.486c.2 0 .4-.1.5-.2l1.8-1.8c.3-.3.3-.8 0-1-.3-.2-.8-.3-1 0l-1.8 1.8c-.3.3-.3.8 0 1 .1.1.3.2.5.2z"
+                                transform="matrix(.88398 0 0 .88643 .396 -16.506)" />
+                        </g>
+                        <g transform="translate(-3.77)" fill="#fff">
+                            <circle cx="8.439" cy="23.002" r="1.015" />
+                            <circle cx="8.439" cy="3.724" r="1.015" />
+                            <path d="M9.453 7.782a1.014 1.014 0 00-2.029 0v11.161a1.015 1.015 0 002.029 0V7.782z" />
+                        </g>
+                    </svg>
+
+                    <label for="strength">Strength</label>
+                    <select name="" id="strength">
+                        <option value="testing">Testing A Thing</option>
```

Comment:
The `option`s here should be updated based on the comp and example provided [here](https://github.com/mica-web/gd2/blob/1fa66b0aaa4ab516217ec9702adf905c198c187f/project3/index.html#L45).